dpdata/unit.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABC
+
+AVOGADRO = 6.02214179E23    # Avagadro constant
+ELE_CHG  = 1.602176487E-19  # Elementary Charge, in C
+
+# energy conversions
+econvs = {
+    "eV": 1.0,
+    "hartree": 4.35974394E-18 / ELE_CHG,
+    "kJ_mol": 1 / (ELE_CHG * AVOGADRO / 1000),
+    "kcal_mol": 1 / (ELE_CHG * AVOGADRO / 1000 / 4.184),
+    "rydberg": 2.17987197E-18 / ELE_CHG,
+    "J": 1 / ELE_CHG,
+    "kJ": 1000 / ELE_CHG
+}
+
+# length conversions
+lconvs = {
+    "angstrom": 1.0,
+    "bohr": 0.52917720859,
+    "nm": 10.0,
+    "m": 1E10,
+}
```

Comment:
We may need to confirm if `4.35974394E-18`, `2.17987197E-18`, and `0.52917720859` is the latest value or not.

Reply:
I agree. These constants should be provided with as many digits as possible.

Reply:
I think we need to have a unified reference. The constant given here is from _**CRC Handbook of Chemistry and Physics**_ (90th edition), which is published in 2010. I'll look up for a latest data version